web/src/Components/CombinationsWrapper.tsx

```diff
@@ -92,6 +96,22 @@ export default ({ products }: Products) => {
     setCombinations({ ...combinations, [combination.name]: combination })
   }
 
+  async function addCombinationsFromOptimization(sackCombination: Combination, densityCombination: Combination) {
+    if (sackCombination.values && densityCombination.values) {
+      const newCombinations: Combinations = {
+        ...combinations,
+        [sackCombination.name]: sackCombination,
+        [densityCombination.name]: densityCombination,
+      }
+      setCombinations(newCombinations)
+      await fetchAllBridges(newCombinations).then(_bridges => {
```

Comment:
solved by calling fetchAllBridges() with argument being the two new combinations from the optimization, instead of letting the argument be all combinations.